docs/rfcs/XeTile.md

```diff
@@ -140,6 +121,79 @@ A `tile_mma` variant without vector_c initialization.
 		tile<64x64xbf16>, index, index into tile <64x64xbf16>
 ```
 
+
+`atomic_rmw_tile` atomically reads, modifies, and writes back data to the memory specified by the tile.
+
+```mlir
+  %ret_value = XeTile.atomic_rmw “addf” %value, %tile:
```

Comment:
can use "\<addf\<" to keep format consistent with the one in table.

Reply:
fixed